Classes/Driver/Version2/Query/FilteredQuery.php

```diff
@@ -18,4 +18,16 @@
  */
 class FilteredQuery extends Version1\Query\FilteredQuery
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function fulltext($searchWord)
+    {
+        $this->appendAtPath('query.filtered.query.bool.must', [
+            'query_string' => [
+                'query' => $searchWord,
+                'fields' => ['__fulltext*']
```

Comment:
This is quiet breaking and hard to detect for existing project, maybe `['_all', '__fulltext*']` is less breaking, should just improve the score of the matching string in `_all` and `__fulltext*`

Reply:
Not sure if highlights work in this condition, but it should

Reply:
It would work.